definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-alt-service-process-server-nonprod.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "LiveFrom": "05/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "ProcessServerServiceCaseField",
+    "UserRole": "caseworker-divorce-courtadmin",
+    "CRUD": "R"
```

Comment:
Are you sure all users should only have "R"? How will the "yes" value be written?

Reply:
They shouldnt. This was just a preliminary filler because at the time, there were no permissions set. Thanks for the good eye! Updating now